src/bin/repl/commands.rs

```diff
@@ -0,0 +1,26 @@
+use super::Repl;
+use std::collections::HashMap;
+
+pub fn default_commands() -> HashMap<&'static str, fn(&mut Repl, &str)> {
+    let mut map = HashMap::<&'static str, fn(&mut Repl, &str)>::new();
+    map.insert("help", help_command);
+    map.insert("quit", quit_command);
+    map.insert("q", quit_command);
+    map
+}
+
+fn help_command(_repl: &mut Repl, _args: &str) {
+    println!(
+        "
+Available commands:
+    {p}help      Shows this message
+    {p}quit|q    Quits the repl
+",
```

Comment:
This prints an extra newline at the start and end.
```suggestion
    print!(
        "\
Available commands:
    {p}help      Shows this message
    {p}quit|q    Quits the repl
",
```